Code/Framework/AzTest/AzTest/UnitTest.h

```diff
@@ -10,12 +10,6 @@
 #include <AzTest/AzTest.h>
```

Comment:
I need a bit to mull over this change, particularly in regards to enshrining the UnitTest namespace as an example to other modules.

Reply:
Interesting, the UnitTest header/namespace had already been injected into AzTest.

Reply:
I'm okay with this dependency moving into AzTest, though not most of the other files